src/pages/ComposePage/index.jsx

```diff
@@ -0,0 +1,12 @@
+import EmailForm from "../../components/EmailForm"
+
+function ComposePage(){
+
+    return(
+        <div className="flex flex-col">
```

Comment:
Same as my comment on the SentPage component - this doesn't do anything apart from display the EmailForm.

Remove this one, move EmailForm into the pages folder and rename it accordingly. Then plug it directly into the route.